knowledge-base/grid-compact-grid-with-small-elements.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Make Compact Grid Elements Smaller
+description: How to make all elements in a Compact Grid smaller.
+type: how-to
+page_title: Make Compact Grid elements smaller
+slug: grid-compact-grid-with-small-elements
+position: 
+tags: telerik, blazor, grid, compact, elements
+ticketid: 1650305
+res_type: kb
+---
+
+# Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+I am using [the Grid sizing feature]({%slug grid-sizing%}) and my Grid is a Compact Grid. I want to reduce the size of all elements in the Grid. How can I:
+* Change the size of all icons in the Grid?
+* Set smaller font size to all elements in the Grid?
+
+## Solution
+
+1. Set [the Grid `Class` parameter]({%slug grid-overview%}#grid-parameters) to a custom CSS class. This allows you to target specific Grid instance.
+1. Use the defined class to [override the theme styles]({%slug themes-override%}).
+1. Set the required CSS styles to the desired elements in the Grid.
```

Comment:
These two steps can be merged into one, because we are practically saying the same thing.